client/components/Landing/LandingStats/CounterCard/CounterCard.module.scss

```diff
@@ -0,0 +1,33 @@
+@import "../../../../styles/variables";
+
+.counter-card {
+  width: 11em;
+  .icon-box {
+    width: 5em;
+    height: 5em;
+    margin: 1em auto;
+    border: 2px solid color(white);
+    transform: rotate(45deg);
+    transition: transform 2s ease-out 100ms;
+    &:hover {
```

Comment:
It feels a bit strange to rotate the entire icon * rectangle.

1. Let's rotate just the white rectangle (without the icon) and just to `90deg` so it's straight.
2. Let's shorten the transition time. Make it smaller so it feels more responsive